apps/frontend/src/components/global/upload_tabs/tenable/FileList.vue

```diff
@@ -0,0 +1,262 @@
+<template>
+  <span>
+    <div class="d-flex flex-row-reverse" style="cursor: pointer">
+      <v-btn icon @click="updateSearch">
+        <v-icon
+          b-tooltip.hover
```

Comment:
isn't this b-tooltip stuff like bootstrap specific functionality?  we're not using bootstrap here

Reply:
it works